packages/notebook/src/browser/view/notebook-main-toolbar.tsx

```diff
@@ -152,17 +152,17 @@ export class NotebookMainToolbar extends React.Component<NotebookMainToolbarProp
 
     override render(): React.ReactNode {
         const menuItems = this.getMenuItems();
-        return <div className='theia-notebook-main-toolbar'>
+        return <div className='theia-notebook-main-toolbar' id='notebook-main-toolbar'>
```

Comment:
Is it save to use a static `id` here? Couldn't there be more than one toolbars in the DOM, eventually leading to non-unique IDs in the DOM? If yes, it might be better to use `data-<xyz>` attributes and locate them in the context of a parent?

Reply:
I think it is okay to use a static `id` for the **main** toolbar. Should be only one, right? In case there are other sibling toolbars with the same `id` it can be handled as a list. For react one will need to adjust the `key` property.

Reply:
There is one toolbar **per editor**, meaning that it's not unique. I.e. if you have two notebook editors side by side, it will no longer be unique.

Reply:
They will be unique if using `locator` API instead of `selector`, what I do in notebook editor page object.  Locator gives you the Notebook editor element under test, hence all subsequent queries are applied on the editor element and not the whole page.